ixmp/core.py

```diff
@@ -69,10 +79,13 @@ class Platform(object):
         the type of the local database (e.g., 'HSQLDB')
         if no 'dbprops' is specified, the local database is
         created/accessed at '~/.local/ixmp/localdb/default'
+    jvmargs : string
+        the allocated max heap space for the java virtual machine
+        eg.: "-Xmx4G"
```

Comment:
Final note, can you add a link to the Java CLI docs: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/java.html so users can look up themselves other options? Then that's it for me!